### PR TITLE
create a server that responds to a hardcoded string when a client requests it URL and change width of website

### PR DESCRIPTION
**Pull request by ammr-mo**
create a server that responds to a hardcoded string when a client requests it URL and change width of website

### CODE REVIEW

File: .theia/settings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (cort-p) [inline .theia/settings.json]: Curious - what is this file doing?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -26,3 +26,11 @@ function addRandomPoem() {
   const poemContainer = document.getElementById('poem-container');
   poemContainer.innerText = poem;
 }
+
+async function showInterest(){
```
Comment (cort-p) [inline portfolio/src/main/webapp/script.js]: Minor nit: add a space before the opening brace.
ie. `async function showInterest() {`

**Review by cort-p — APPROVED** [review]

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -12,6 +12,9 @@ <h1>Ammran Mohamed's Portfolio</h1>
       <p>This is my portfolio.</p>
       <img src="/images/selfie.jpg" width="300" height="400"/>
       <p> My name is Ammran Mohamed and I am a junior at Columbia University studying Computer Science. I am particularly interested in Artificial Intelligence and Machine Learning.</p>
+      <p> Click this button to learn more about what I am interested in! </p>
+      <button onclick="showInterest()">?????</button>
```
Comment (sarahisyoung) [inline portfolio/src/main/webapp/index.html]: don't forget to change '???'

File: .theia/settings.json (hunk 1 above)
Comment (ammr-mo) [reply .theia/settings.json]: I'm not sure, don't remember making it...

**Pull request merged**